test/test_helper.rb

```diff
@@ -108,16 +108,16 @@ def add_items_to_cart
 
   def create_categories_items_user_order_and_login
     create_categories_and_items
-    current_user = User.create(username: "Matt", password: "gnargnar")
+    current_user = User.create(username: 'Matt', password: 'gnargnar')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.